build-logic/src/main/kotlin/com.hedera.hashgraph.java.gradle.kts

```diff
@@ -0,0 +1,174 @@
+/*
+ * Copyright (C) 2022 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
```

Comment:
This is moved up from `com.hedera.hashgraph.conventions` as a shared base for
- `com.hedera.hashgraph.conventions` (projects in 'hedera-node')
- `com.hedera.hashgraph.sdk.conventions` (projects in 'platform-sdk')